packages/react-devtools-extensions/src/background/index.js

```diff
@@ -64,8 +64,8 @@ chrome.runtime.onConnect.addListener(port => {
     const tabId = port.sender.tab.id;
 
     if (ports[tabId]?.proxy) {
-      port.disconnect();
-      return;
+      ports[tabId].disconnectPipe?.();
+      ports[tabId].proxy.disconnect();
```

Comment:
So, this `proxy` port is opened from content script in isolated environment, which has access to `window` object, but not DOM.

This condition is usually an edge case, it means that previous port was not disconnected or we were not notified that it was disconnected. In previous implementation we would disconnect new port, which is questionable, but it could save us from situation with many new connections spam.

I think better approach in this case is disconnecting previous port properly and registering new one, connecting it with devtools page (in browser DevTools) port.